tests/rootfs.sh

```diff
@@ -12,7 +12,7 @@ RESULT=0
 
 setup()
 {
-    if [ ! -x "${ARM_EMU_BIN}" ]; then
+    if [ ! -x "${ARM_EMU_BIN}" ] || [ "${ARM_EMU_BIN#/}" != "${ARM_EMU_BIN}" ]; then
```

Comment:
Also it is nowhere stated in the usage of these script when and what to pass as env with the scripts